maltoolbox/attackgraph/attackgraph.py

```diff
@@ -493,6 +333,176 @@ def attach_attackers(self) -> None:
 
             attacker.entry_points = list(attacker.reached_attack_steps)
 
+    def _follow_expr_chain(
+            self,
+            model: Model,
+            target_assets: list[Any],
```

Comment:
Is this really `list[Any]`? Isn't it always:

```suggestion
            target_assets: list[LanguageGraphAsset],
```
?

Reply:
These are instance model assets, not language graph asset types.

Reply:
Aha, so this will have to change if we remove the class factory and instead of one class per model asset we have instances of a single ModelAsset class.